.github/workflows/test-build-deploy.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: Upgrade golang
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17.8
+          go-version: 1.18.x
```

Comment:
why not 1.19.X?

Reply:
OK I don't want to update too many things since Thanos is still at 1.18.X so I think 1.18.X is good here.
1.19 is also okay, let me update.